cmds/webboot/webboot.go

```diff
@@ -108,3 +158,28 @@ func download(URL, fPath string) error {
 	}
 	return nil
 }
+
+// getHierachyMenu makes a hierarchy menu:
+// level 1: 0.Install cached ISO;1.Download ISO by bookmark; 2.Download ISO by link
```

Comment:
I think in this comment its enough to just say what the 3 options are, and mention that the menu will display nested levels of directories until an ISO is chosen, or something along those lines. Lets try to keep the comments short if we can. The specifics of each option can be talked about in the comments for those specific functions.